### PR TITLE
avm2: Stub flash.globalization.DateTimeFormatter

### LINKED ISSUE #14577: error at https://sites.google.com/view/super-math-game/platform-racing-2?authuser=0
### Describe the bug

on platform racing 2 fails to load past the first screen

image_2024-01-01_135940070

### Expected behavior

image_2024-01-01_135908052

### Content Location

https://sites.google.com/view/super-math-game/platform-racing-2?authuser=0

### Affected platform

Self-hosted version

### Operating system

Windows 10

### Browser

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by n0samu**
Closes #14577

### CODE REVIEW

**Comment by torokati44** [comment]
Should parameter null checking not be done in all methods?

**Comment by n0samu** [comment]
Not important, but yes. I added it to the remaining methods.

**Review by torokati44 — APPROVED** [review]

File: core/src/avm2/globals/flash/globalization/DateTimeFormatter.as [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package flash.globalization {
+    import __ruffle__.stub_constructor;
+    import __ruffle__.stub_getter;
+    import __ruffle__.stub_method;
+    import flash.globalization.LastOperationStatus;
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/globalization/DateTimeFormatter.as:5]: This import is unnecessary

Comment (torokati44) [reply core/src/avm2/globals/flash/globalization/DateTimeFormatter.as:5]: It's used on line 32.

Comment (torokati44) [reply core/src/avm2/globals/flash/globalization/DateTimeFormatter.as:5]: If anything, I was wondering whether the string literal default parameter values should be their corresponding named constants ("enums", like this one) instead. But the AS3 docs also say string literals there, so...

Comment (n0samu) [reply core/src/avm2/globals/flash/globalization/DateTimeFormatter.as:5]: I tried that already and the AS3 compiler doesn't allow it.

**Pull request merged**